snapcraft/internal/cache/_snap.py

```diff
@@ -76,6 +80,20 @@ def prune(self, *, keep_revision):
                         'Unable to purge snap {}.'.format(cached_snap))
         return pruned_files_list
 
+    def get_latest(self, snap_name):
```

Comment:
can this be `get`, also defined in `_cache` and isn't `project_name` the `snap_name` already?

```yaml
def get(hash=None):
    """Get the revision determined in hash or the latest cached item."""
```